src/transforms/ppmt.jl

```diff
@@ -0,0 +1,171 @@
+using EmpiricalCDFs
+using Distributions
+using Optim
+
+"""
+    Ppmt()
```

Comment:
The convention is that transform names are all UPPERCASE when they are acronyms. Please rename to PPMT.